content/javascript/concepts/iterators/iterators.md

```diff
@@ -0,0 +1,146 @@
+---
+Title: "Iterators" 
+Subjects: 
+  - "Web Development"
+  - "Computer Science"
+Tags:
+  - "Iterators"
+CatalogContent: 
+  - "introduction-to-javascript"
+  - "paths/front-end-engineer-career-path"
+---
+
```

Comment:
Let's add a sentence or two here about what is an iterator in JavaScript and what it's for.